llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -649,6 +649,10 @@ cc_library(
     deps = [
         ":Demangle",
         ":config",
+        # We unconditionally depend on the custom LLVM zlib wrapper. This will
+        # be an empty library unless zlib is enabled, in which case it will
+        # both provide the necessary dependencies and configuration defines.
+        "@llvm_zlib//:zlib",
```

Comment:
Shouldn't this be part of config instead? That's where the zlib defines are usually, I think?

Reply:
Ok. Wasn't sure if we needed the defines higher up, since they're usually defined in config.h. If that was the case, we could have separate zlib and zlib_defines rules.